packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -64,7 +60,8 @@ const solutionLevelParameterObject = {
 const templateFolderNew = "./templates/azure";
 const configsFolder = `.${ConfigFolderName}/configs`;
 const modulesFolder = "modules";
-const parameterFileNameTemplateNew = "azure.parameters.@envName.json";
+const parameterFileNameTemplate = "azure.parameters.@envName.json";
+const parameterFileDefaultTemplate = "azure.parameters.dev.json";
```

Comment:
How about calling `environmentManager.getDefaultEnvName()` to get the default env name instead of the hard-coded `dev`?

Reply:
sure, thanks!

Reply:
I didn't test it but I guess `environmentManager.getDefaultEnvName()` (which uses `isMultiEnvEnabled()`) may not work in global scope because `syncFeatureFlags()` is called after extension's `activate()`

Reply:
The logic here does not use this value, it has been removed